_data/simple-icons.json

```diff
@@ -7848,6 +7848,11 @@
             "hex": "000000",
             "source": "https://jwt.io"
         },
+        {
+            "title": "JSR",
+            "hex": "F7DF1E",
```

Comment:
This is kinda tricky, because while `#f7df1e` is indeed the correct _internal_ color for the logo, this property actually sets the color of the outer "stroke," which should be `#083344`.

I don't know that there's a better solution for a monochromatic SVG though, honestly. Users implementing this version of the logo will, presumably, just want one color anyway.

Reply:
Seeing as that shade of yellow is the primary color of the logo that's what we'll go with, though you are correct that it sets the color of a different part of the SVG here.

Reply:
@josh-collinsworth This below should be a workaround:

```
https://img.shields.io/badge/JSR-1.2.0-083344?logo=npm&logoColor=083344&labelColor=f7df1e
https://img.shields.io/badge/1.1.0-083344?logo=npm&logoColor=083344&labelColor=f7df1e
```

![](https://img.shields.io/badge/JSR-1.2.0-083344?logo=npm&logoColor=083344&labelColor=f7df1e)
![](https://img.shields.io/badge/1.1.0-083344?logo=npm&logoColor=083344&labelColor=f7df1e)

I used the npm logo as an example since your logo has not been merged yet.

Also, I have a PR for wide shapes on https://sheilds.io. See https://github.com/badges/shields/pull/9191. So your JSR logo will display nicely on the badge.